presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/test/RuleAssert.java

```diff
@@ -138,36 +142,38 @@ public void matches(PlanMatchPattern pattern)
 
         inTransaction(session -> {
             Map<PlanNodeId, PlanNodeCost> planNodeCosts = costCalculator.calculateCostForPlan(session, types, actual);
-            assertPlan(session, metadata, costCalculator, new Plan(actual, types, planNodeCosts), ruleApplication.lookup, pattern);
+            assertPlan(session, metadata, costCalculator, new Plan(actual, types, planNodeCosts), result.lookup, pattern);
             return null;
         });
     }
 
-    private RuleApplication applyRule()
+    private OptimizationResult optimize()
```

Comment:
calling it "optimize" is not optimal, since rule are not, by definition, optimizations. Some are, but some aren't. Previously this method used to be called "applyRule" and I think this is much better name.